src/utils.js

```diff
@@ -145,13 +147,15 @@ export const validateAlphaNumericField = value => {
 
 export const checkIfElementIsEmpty = element => (element === '-' ? noValue : element);
 
+export const checkIfArrayIsEmpty = array => (!isEmpty(array) ? array : emptyList);
+
 export const convertArrayToBlocks = elements => (!isEmpty(elements)
-  ? elements.map((line, i) => <div key={i}>{line}</div>)
-  : '-');
+  ? elements.map((line, i) => (line !== null ? <div key={i}>{line}</div> : noValue))
```

Comment:
Maybe it makes sense to check just line? If the line will be equal to undefined or empty value, this check will not pass. 
```suggestion
  ? elements.map((line, i) => (line ? <div key={i}>{line}</div> : noValue))
```